### PR TITLE
feat(cohort creation): search condition VS by code and edit VS url in config

### LINKED ISSUE #93: As a user I can see the diagnosis codes and search them when adding a diagnosis cohort criteria
## Problème
Dans l'ajout de critères pour créer une cohorte, dans PMSI> Diagnostics, il faudrait pouvoir choisir parmi les codes de la CIM-10, (aujourd'hui CIM-9 uniquement). Il faudrait aussi pouvoir rechercher par code et pas par libellé uniquement.  

## Description
Ce que ça implique : 
- Créer/ajouter un ValueSet CIM-10, et peut être un ValueSet qui regroupe CIM-10, CIM-9 @nriss tu en penses quoi?
- Modifier l'interface : pouvoir faire le choix de la classif? Ou bien avoir une barre de recherche regroupant les deux ensembles de codes? 
- Modifier le texte affiché (aujourd'hui il y a écrit CIM-9 au niveau de l'ajout de critère, puis CIM-10 une fois le critère ajouté)
- Pouvoir rechercher un diagnostic CIM-* en fonction de son code, ou de son libellé (aujourd'hui uniquement possible avec le libellé, pas le code)
- (Typo) Diagnosti**c** pour le substantif et diagnosti**que** pour l'adjectif

## Contexte
Remarques soulevées par @Peltier10

### PR DESCRIPTION
**Pull request by tevariou**
## Fixes

Fixes #93, #94 and #99  by @elsiehoffet-94, @Peltier10 and @MiskoG

### CODE REVIEW

**Comment by MiskoG** [comment]
Checked it with you Teva, good for me !

**Review by MiskoG — APPROVED** [review]

**Review by nriss — APPROVED** [review]
🚀

**Review by elsiehoffet-94 — APPROVED** [review]
Thanks a lot!

**Review by simonvadee — APPROVED** [review]

File: src/services/cohortCreation/fetchCondition.ts [hunk 1]
```diff
@@ -86,26 +84,24 @@ type Code = {
...
-    if (!valueSet || valueSet.length === 0) return []
-    return (
-      valueSet[0]?.compose?.include[0]?.concept
-        ?.map((value) => ({
-          id: value.code,
-          label: value.display
-        }))
-        .sort((a, b) => (a.label && b.label ? a.label.localeCompare(b.label) : 0)) ?? []
-    )
+    const { data: valueSet } = await api.get<IValueSet>(`/ValueSet/$expand?url=${CONDITION_VS_URL}`)
```
Comment (tevariou) [inline src/services/cohortCreation/fetchCondition.ts]: @nriss a little warning here. It won't work if the `$expand` doesn't or if it's not needed (if the values are in `valueSet.compose.include[x].concept` as before for example).

Comment (nriss) [reply src/services/cohortCreation/fetchCondition.ts]: I tried on this vs : ValueSet/$expand?url=http://hl7.org/fhir/ValueSet/ucum-bodylength and got the two defined codes in expansion

I think the problem comes from the VS definition

**Pull request merged**